testing/unit_tests/test_rgb.py

```diff
@@ -2,33 +2,30 @@
 import numpy as np
 import torch
 import os
-from testing.unit_tests.t_utils import create_tmp_dir,  __data_testing_dir__, __tmp_dir__
+from testing.unit_tests.t_utils import create_tmp_dir,  __data_testing_dir__, __tmp_dir__, download_data_testing_test_files
 from testing.common_testing_util import remove_tmp_dir
 
 def setup_function():
     create_tmp_dir()
 
 
-# TODO (#720): un-silence this test. This test was silenced because it was failing after introducing the copy of the header in #714.
-#  It might be possible to solve the issue with the test by initializing the "image" with a proper shape, i.e. one that
-#  would match that of the prediction.
-# def test_save_rgb():
-#     # Create image with shape HxWxDxC
-#     image = [[[[0 for i in range(10)] for i in range(10)] for i in range(4)] for i in range(3)]
-#     for i in range(3):
-#         image[0][2][5][i] = 1
-#         image[1][2][5][i+3] = 1
-#         image[2][2][5][i + 6] = 1
-#     image_n = np.array(image)
-#     imed_visualize.save_color_labels(
-#         gt_data=image_n,
-#         binarize=False,
-#         gt_filename=os.path.join(
-#             __data_testing_dir__,
-#             "derivatives/labels/sub-unf01/anat/sub-unf01_T2w_lesion-manual.nii.gz"),
-#         output_filename=os.path.join(__tmp_dir__, "rgb_test.nii.gz"),
-#         slice_axis=0
-#     )
+def test_save_rgb(download_data_testing_test_files):
+    # Create image with shape HxWxDxC
+    image = [[[[0 for i in range(10)] for i in range(10)] for i in range(4)] for i in range(3)]
+    for i in range(3):
+        image[0][2][5][i] = 1
+        image[1][2][5][i+3] = 1
+        image[2][2][5][i + 6] = 1
+    image_n = np.array(image)
+    imed_visualize.save_color_labels(
+        gt_data=image_n,
+        binarize=False,
+        gt_filename=os.path.join(
+            __data_testing_dir__,
+            "rgb_test_file.nii.gz"),
```

Comment:
This input reference image I presume already has the correct header. Could you maybe briefly elaborate a bit about how that was created initially for record purposes? For the purpose of the test of "saving" functionalities, it is more than sufficient. From a test perspective, we achieved the goal if restoring the earlier test and ensure the `save_color_labels` are correctly saving the files inputs are correct.

Reply:
this new image file is created by using save_color_labels function without providing a header for the nifti image, I am just using it again to copy the header